### PR TITLE
Add HeliChain

### PR DESCRIPTION
**Pull request by HeatApp-Corp**
HeliChain is a groundbreaking blockchain Layer 1 solution, driven by our exceptional team. With a decentralized approach, we strive to connect people with the web3 world, serving as the gateway to this transformative realm.

### CODE REVIEW

File: heli/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "helichain",
```
Comment (JeremyParish69) [inline heli/assetlist.json:3]: This chain_name must match the directory name, so either chain_name has to change to 'heli' or the directory name must change to 'helichain'. This value is immutable.

File: heli/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        {
+          "denom": "heli",
+          "exponent": 6
+        }
+      ],
+      "base": "uheli",
+      "name": "Heli",
+      "display": "heli",
+      "symbol": "HELI",
+      "logo_URIs": {
```
Comment (JeremyParish69) [inline heli/assetlist.json:21]: Please omit the `logo_URIs` property. (But keep `images`)

File: heli/chain.json [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+          "type": "cometbft",
+          "version": "0.37.2"
+        },
+        "cosmwasm_version": "0.42.0",
+        "cosmwasm_enabled": true,
+        "next_version_name": ""
+      }
+    ]
+  },
+  "logo_URIs": {
```
Comment (JeremyParish69) [inline heli/chain.json:73]: please omit this `logo_URIs` property

File: heli/assetlist.json (hunk 1 above)
Comment (HeatApp-Corp) [reply heli/assetlist.json:3]: I updated chain_name

File: heli/assetlist.json (hunk 2 above)
Comment (HeatApp-Corp) [reply heli/assetlist.json]: I removed logo_URIs in assetlist.json

File: heli/chain.json (hunk 3 above)
Comment (HeatApp-Corp) [reply heli/chain.json]: I removed logo_URIs in chain.json

File: heli/chain.json [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+      "kind": "ping.pub",
+      "url": "https://explorer.helichain.com/Helichain",
+      "tx_page": "https://explorer.helichain.com/Helichain/tx/${txHash}",
+      "account_page": "https://explorer.helichain.com/Helichain/account/${accountAddress}"
+    }
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/heli/images/heli.png",
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/heli/images/heli.svg"
```
Comment (JeremyParish69) [inline heli/chain.json:116]: The png and svg you have here are not identical. One has transparect background, while the other has a white background. Please make sure they are the same, or else register them as separate images.

File: heli/chain.json [hunk 5]
```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "heli",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://helichain.com/",
+  "pretty_name": "HeliChain",
+  "chain_id": "helichain",
+  "bech32_prefix": "heli",
```
[the remark this answers is not in the document]
Comment (JeremyParish69) [reply heli/chain.json:9]: Pending https://github.com/satoshilabs/slips/pull/1745/files

[the remark this answers is not in the document]
Comment (HeatApp-Corp) [reply heli/chain.json:9]: #https://github.com/satoshilabs/slips/pull/1745

[the remark this answers is not in the document]
Comment (HeatApp-Corp) [reply heli/chain.json:9]: Thank you very much!

Registered & awaiting merge #https://github.com/satoshilabs/slips/pull/1745

**Pull request merged**